### ISSUE TITLE
No softmax in densenet

### ISSUE THREAD

**Issue by grudloff** [Project-MONAI/tutorials#1520]
I was checking the implementation of densenet and I noticed that the class layers, defined as:
```python
  self.class_layers = nn.Sequential(
      OrderedDict(
          [
              ("relu", get_act_layer(name=act)),
              ("pool", avg_pool_type(1)),
              ("flatten", nn.Flatten(1)),
              ("out", nn.Linear(in_channels, out_channels)),
          ]
```

There is no softmax in the output, and I find this odd. Am I missing something?

**Comment by KumoLiu** [comment]
Hi @grudloff, I think you can add it by using `Activations`.
https://github.com/Project-MONAI/MONAI/blob/281cb0119c01eaa8e6c841880b91f92f45e8d7f7/monai/transforms/post/array.py#L59

Thanks!
